src/main/java/command/Command.java

```diff
@@ -0,0 +1,61 @@
+package main.java.command;
+
+import main.java.components.Storage;
+import main.java.components.TaskList;
+import main.java.exception.EmptyDescriptionException;
+import main.java.exception.InvalidDescriptionException;
+import main.java.exception.InvalidInputException;
+import java.io.IOException;
+import java.text.ParseException;
+
+/**
+ * An abstract Command class with abstract method execute and isExit.
+ * This Command class handles the command given by the user.
+ */
+public abstract class Command {
```

Comment:
Good Job! 

I like the implementation of OOP in this class!

Reply:
Thank you!